src/app/client/src/app/modules/public/module/signup/components/signup/signup.component.scss

```diff
@@ -210,4 +210,7 @@ input[type=number]::-webkit-inner-spin-button {
         font-weight: normal;
         cursor: pointer;
       }
+  }
```

Comment:
please remove the css changes as we will not be using rem for calculating the size

If need please create a task for the UI Team and get their help to fix the UI

Reply:
Done